### PR TITLE
Functional tests - Add new test : CRUD Seo Page

### PR DESCRIPTION
**Pull request by boubkerbribri**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Add new functional test : CRUD Seo Page
| Type?         | refacto
| Category?     | TE
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | no
| How to test?  | `TEST_PATH="functional/BO/shopParameters/trafficAndSeo/seoAndUrls/02_CRUDSeoPage.js" URL_FO=Shop_URL npm run specific-test`

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/16959)

### CODE REVIEW

**Comment by nesrineabdmouleh** [comment]
Thank you @boubkerbribri :)

File: tests/puppeteer/pages/BO/shopParameters/trafficAndSeo/seoAndUrls/add.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+  }
+
+  /**
+   * Delete all keywords
+   * @param lang, to specify which input to empty
+   * @return {Promise<void>}
+   */
+  async deleteKeywords(lang = 'en') {
+    const closeButtons = await this.page.$$(this.deleteKeywordLink.replace('%LANG', lang));
+    /* eslint-disable no-restricted-syntax */
```
Comment (NeOMakinG) [inline tests/puppeteer/pages/BO/shopParameters/trafficAndSeo/seoAndUrls/add.js]: disable this rule or replace it by a foreach maybe ?

File: tests/puppeteer/pages/BO/shopParameters/trafficAndSeo/seoAndUrls/add.js [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+  }
+
+  /**
+   * Add keywords
+   * @param keywords, array of keywords
+   * @param idLang, to choose which lang (1 for en, 2 for fr)
+   * @return {Promise<void>}
+   */
+  async addKeywords(keywords, idLang = 1) {
+    /* eslint-disable no-restricted-syntax */
```
Comment (NeOMakinG) [inline tests/puppeteer/pages/BO/shopParameters/trafficAndSeo/seoAndUrls/add.js]: same here

**Review by NeOMakinG — CHANGES_REQUESTED** [review]

File: tests/puppeteer/pages/BO/shopParameters/trafficAndSeo/seoAndUrls/add.js (hunk 1 above)
Comment (boubkerbribri) [reply tests/puppeteer/pages/BO/shopParameters/trafficAndSeo/seoAndUrls/add.js]: we want to click on buttons in sequence (in order to click on the right element), we can't do that with foreach (already tested).

Comment (boubkerbribri) [reply tests/puppeteer/pages/BO/shopParameters/trafficAndSeo/seoAndUrls/add.js]: replaced by 
```js
for (let i = 0; i < closeButtons.length; i++)
```

**Review by NeOMakinG — APPROVED** [review]

**Pull request merged**